daemon/policy.go

```diff
@@ -252,11 +252,12 @@ func (d *Daemon) policyAdd(sourceRules policyAPI.Rules, opts *AddOptions, resCha
 	// CAUTION, there is a small race between this PrepareFQDNRules invocation and
 	// taking the policy lock. As long as policyAdd is fed by a single-threaded
 	// queue this should never be an issue.
-	rules := d.dnsRuleGen.PrepareFQDNRules(sourceRules)
+	// TODO: ian fix me
```

Comment:
What needs fixing here?

Reply:
+1

Reply:
Fixed with more descriptive comment - basically we are not pre-populating the selectorcache in this case. I think this is OK to defer for now.